Update README.md example to match version 0.7.0 (#1750)
linebender/druid: README.md

    Flex::column().with_child(label).with_child(button)
}
```

> ##### This example is compatible with master, [here's an example using 0.7.0](https://crates.io/crates/druid/0.7.0#example)
> <br>
> 

Reviewer: ```suggestion
This example works with master. Here's an example for 0.7.0: https://docs.rs/druid/latest/druid/#examples.
```
